packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -38,9 +38,10 @@ interface SnapshotProposalIpfs {
   plugins: SafeSnapPlugin;
 }
 
-// We only type the properties requested in the GraphQL query. This extends SnapshotProposalIpfs, but we need to
+// We only type the properties requested in the GraphQL queries. This extends SnapshotProposalIpfs, but we need to
 // override the space property.
-interface SnapshotProposalGraphql extends Omit<SnapshotProposalIpfs, "space"> {
+export interface SnapshotProposalGraphql extends Omit<SnapshotProposalIpfs, "space"> {
+  id: string;
```

Comment:
id is not needed for proposal verification, but we keep it here to use the same type as in proposer bot where id is required